src/renderer/coremods/settings/pages/General.tsx

```diff
@@ -1,23 +1,94 @@
 import React from "@common/react";
+import { Divider, Flex, FormItem, SwitchItem, Text, TextInput } from "@components";
+import { settings, util } from "@replugged";
+import { Jsonifiable } from "type-fest";
+
+export interface GeneralSettings {
+  apiUrl: string;
+  // pluginEmbeds: boolean;
+  experiments: boolean;
+  [key: string]: Jsonifiable;
+}
+const defaultSettings: Partial<GeneralSettings> = {
+  apiUrl: "https://replugged.dev",
+  // pluginEmbeds: false,
+  experiments: false,
+};
+const configs = await settings.init<GeneralSettings, keyof typeof defaultSettings>(
```

Comment:
Export this and use this in the other parts of the app instead of `init`ing it each time. More efficient, plus you won't have to put fallbacks for each `.get`.